src/hooks/apis/useDeerSighting.ts

```diff
@@ -10,6 +11,8 @@ export const createDeerSighting = async (newSighting: DeerSighting) => {
   });
 
   if (!response.ok) {
+    console.log('error response:');
+    console.log(response.json());
```

Comment:
Same as above, we might wanna consider saving errors to some type of exception log in the future

Reply:
mentioned this in a previous comment but `console.error` will be used going forward to log errors just to raise awareness